packages/dynamic_layouts/README.md

```diff
@@ -25,13 +24,30 @@ TODO: List what your package can do. Maybe include images, gifs, or videos.
 TODO: List prerequisites and provide or point to information on how to
 start using the package.
 
+### Depend on it
+
+Run this command with Flutter:
+```sh
+$ flutter pub add dynamic_layouts
+```
+### Import it
+
+Now in your Dart code, you can use:
+
+```dart
+import 'package:dynamic_layouts/dynamic_layouts.dart';
+```
 ## Usage
 
 TODO: Include short and useful examples for package users. Add longer examples
 to `/example` folder.
 
 ## Additional information
-
-TODO: Tell users more about the package: where to find more information, how to
-contribute to the package, how to file issues, what response they can expect
-from the package authors, and more.
+This package was developed to recreate native scrolling layouts.
```

Comment:
Also, let's remove this line. This package can contain all kinds of scrolling layouts in the future.